### PR TITLE
fixing various issues

### PR DESCRIPTION
**Pull request by sethladd**
Closes  https://github.com/flutter/flutter/issues/3540 
closes https://github.com/flutter/flutter/issues/3538
closes https://github.com/flutter/flutter/issues/3536
closes https://github.com/flutter/flutter/issues/3536
closes https://github.com/flutter/flutter/issues/3533

should also help a little with https://github.com/flutter/flutter/issues/3476

The downgrade of libv8 was purposeful, because it didn't build on mac 10.11 otherwise.

### CODE REVIEW

**Comment by sethladd** [comment]
cc @devoncarew

File: _includes/head.html [hunk 1]
```diff
@@ -2,34 +2,22 @@
...
+  <link rel="stylesheet" href="{{ "/css/modern-business.css" | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ "/css/customstyles.css" | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ "/css/theme-blue.css" | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ "/css/lavish-bootstrap.css" | prepend: site.baseurl }}">
 
+  <link rel="shortcut icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl  }}">
 
-  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro|Roboto:500,400italic,300,400" rel="stylesheet" type="text/css">
-  <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro%7CRoboto:500,400italic,300,400" rel="stylesheet">
```
Comment (devoncarew) [inline _includes/head.html]: intended?

**Comment by devoncarew** [comment]
lgtm, though I'm less familar w/ the site code. Perhaps add a screenshot or two of the visual fixes?

File: _includes/head.html (hunk 1 above)
Comment (sethladd) [inline _includes/head.html]: Yup (calms the validator). Good catch, btw!

**Comment by sethladd** [comment]
screen shot 2016-04-25 at 2 41 37 pm

**Comment by sethladd** [comment]
screen shot 2016-04-25 at 2 42 15 pm

**Comment by sethladd** [comment]
fixes https://github.com/flutter/flutter/issues/3539

**Comment by sethladd** [comment]
Should also address https://github.com/flutter/flutter/issues/3543

**Pull request merged**